lib/Extension/CodeTransformExtra/Rpc/OverrideMethodHandler.php

```diff
@@ -66,7 +66,7 @@ public function handle(array $arguments)
             self::PARAM_METHOD_NAME,
             sprintf('Methods from "%s"', $parentClass->name()),
             $this->methodChoices($parentClass)
-        ));
+        )->setAllowMultipleResults(true));
```

Comment:
`set` and `get` are not used in Phpactor. The "responses" are immutable value objects. I think the convention here for an optional would be `withMultiple(true)` which would return a new instance, see https://github.com/phpactor/rpc-extension/blob/master/lib/Response/OpenFileResponse.php#L85

I'd handle optional values differently next time, but this is the way it's done now...

Reply:
It's not the response that is modified but the `ListInput` instance.
Which is later wrapped into a response with `createInputCallback`.
But if you still prefer an immutable approach I'm fine with that, just let me know.

Reply:
Yes, immutable would be the general approach here, I guess there might be an example? (Away from computer now). But in any case I _think_ what I suggest above would apply